src/utils/installDependencies.ts

```diff
@@ -1,16 +1,16 @@
 import { spawnSync } from 'child_process'
 
-function installDependencies(projectRoot: string, packageManager: 'npm' | 'yarn' | 'pnpm' | 'bun') {
+function installDependencies (projectRoot: string, packageManager: 'npm' | 'yarn' | 'pnpm' | 'bun') {
```

Comment:
If order matters here, I'd suggest 'pnpm', 'yarn', 'npm', 'bun' to match up the docs.